src/main/java/uk/gov/hmcts/reform/sscs/helpers/CaseFlagMappingServiceHelper.java

```diff
@@ -0,0 +1,106 @@
+package uk.gov.hmcts.reform.sscs.helpers;
+
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.sscs.ccd.domain.SscsCaseData;
+
+import java.util.HashMap;
+import java.util.Map;
+
+@Service
+@SuppressWarnings("PMD")
+public class CaseFlagMappingServiceHelper {
+
+    private final String[] mappingValues =
+        {
+        "disabledAccess",
+        "signLanguageType",
+        "hearingLoop",
+        "isConfidentialCase",
+        "dwpUCB",
+        "dwpPHME",
+        "urgentCase",
+        "adjournCaseInterpreterLanguage"
+        };
+
+    private final Map<String, Object> caseFlags;
+
+    public CaseFlagMappingServiceHelper() {
+        this.caseFlags = new HashMap<>();
+    }
+
+    public Map<String, Object> mapCaseFlags(SscsCaseData caseData) {
+        mapSignLanguageType(caseData);
```

Comment:
Moving the mappingValues into the enum would mean you can simply return a list of case flags

Reply:
At this point you may also need mapDisabledAccess(caseData) method. It seems that one is missing

Reply:
> At this point you may also need mapDisabledAccess(caseData) method. It seems that one is missing

its mapped in arrangements